index.html

```diff
@@ -73,68 +103,13 @@ <h2>Main Program</h2>
       </div>
     </div>
     <div class="see-program">
-      <button type="button">See complete program</button>
+      <button type="button">SEE THE COMPLETE PROGRAM</button>
     </div>
   </section>
 
-  <section class="speakers">
+  <section id="speakers" class="speakers">
     <h3 class="featured-title">Featured Speakers</h3>
```

Comment:
- [ ] please like the previous reviewer noted, Kindly make sure to add a separator after the Featured Speaker Headline, so your website looks identical to the provided design
- [ ] please like it was noted, your partners section is wrongly positioned. please fix that
yours
![Screenshot (889)_LI](https://user-images.githubusercontent.com/65298048/226123145-17433171-370d-479e-8f61-26cda03922aa.jpg)
original design
![Screenshot (890)](https://user-images.githubusercontent.com/65298048/226123206-eddcd624-b70e-404b-b0a8-493063569f52.png)